pkg/artifactory/resource/replication/resource_artifactory_pull_replication.go

```diff
@@ -48,19 +76,16 @@ var pullReplicationSchema = map[string]*schema.Schema{
 		Computed: true,
 	},
 	"sync_deletes": {
-		Type:     schema.TypeBool,
-		Optional: true,
-		Computed: true,
```

Comment:
I have a vague recollection about this may be an issue. Meaning once a computed attribute is saved into the TF state, it can't be changed back to non-computed. I need to search the SDK GitHub issues list for this.

Reply:
This is a deprecated resource, we can just revert all computed attributes back, if needed.

Reply:
Let's revert and not make functional changes.

Reply:
Added `Computed` back.